### PR TITLE
docs: fix broken image and change table style

### LINKED ISSUE #411: [chore] substitute all image urls with ones that hosted on githubusercontent.com in case of broken images
### Description

I've been thinking it as an internet issue for so long but now I treat it as a broken image. Maybe we should recapture a screenshot and update it in README.

In README:

image

Open it in seperate tab:

image

Hey @zhicheng-ning, could you please check this issue on your machine? If you can reproduce the problem, we should replace the broken image.

### LINKED ISSUE #413: [docs] gives <table> a <thead> in several markdown files
### Description

image

Files are:

- https://github.com/hypertrons/hypertrons-crx/blob/fea52e15c8090c0cd8ce917b77642cd6e0fbc085/README.md
- https://github.com/hypertrons/hypertrons-crx/blob/fea52e15c8090c0cd8ce917b77642cd6e0fbc085/README.zh-CN.md
- https://github.com/hypertrons/hypertrons-crx/blob/fea52e15c8090c0cd8ce917b77642cd6e0fbc085/INSTALLATION.md
- https://github.com/hypertrons/hypertrons-crx/blob/fea52e15c8090c0cd8ce917b77642cd6e0fbc085/INSTALLATION.zh-CN.md

### PR DESCRIPTION
**Pull request by xgdyp**
## Brief Information

This pull request is in the type of:

- [ ] bug fixing
- [ ] new feature
- [x] docs

Related issues:

- closed #413  
- closed #411

### CODE REVIEW

**Review by xgdyp — COMMENTED** [review]
I think English is better than US

**Review by tyn1998 — APPROVED** [review]
LGTM, thank you so much!

File: INSTALLATION.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Language : US | [zh-CN](./INSTALLATION.zh-CN.md)
+Language : English | [zh-CN](./INSTALLATION.zh-CN.md)
```
[the remark this answers is not in the document]
Comment (xgdyp) [reply INSTALLATION.md]: changed zh-CN to 中文

[the remark this answers is not in the document]
Comment (xgdyp) [reply INSTALLATION.md]: changed zh-CN to 中文

[the remark this answers is not in the document]
Comment (xgdyp) [reply INSTALLATION.md]: changed zh-CN to 中文

[the remark this answers is not in the document]
Comment (xgdyp) [reply INSTALLATION.md]: changed zh-CN to 中文

[the remark this answers is not in the document]
Comment (xgdyp) [reply INSTALLATION.md]: changed zh-CN to 中文

[the remark this answers is not in the document]
Comment (xgdyp) [reply INSTALLATION.md]: changed zh-CN to 中文

[the remark this answers is not in the document]
Comment (xgdyp) [reply INSTALLATION.md]: changed zh-CN to 中文

**Pull request merged**